spec/models/step_spec.rb

```diff
@@ -6,10 +6,6 @@
 
   describe "Validations" do
     it { should validate_presence_of(:proposal) }
-    it do
```

Comment:
It doesn't work - our actual validation allows for blank, and this test throws an error when nil is saved. ?